zkevm-circuits/src/evm_circuit/execution/sstore.rs

```diff
@@ -56,7 +56,7 @@ impl<F: Field> ExecutionGadget<F> for SstoreGadget<F> {
 
         let tx_id = cb.call_context(None, CallContextFieldTag::TxId);
 
-        // constrain not in static call
+        // constraint not in static call
```

Comment:
Nitpick: We can improve the doc string clarity more.
```suggestion
        // Constrain we're not in a STATICCALL context.
```